sb.web.app/package.json

```diff
@@ -59,7 +60,8 @@
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-node": "^1.0.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.0"
+    "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-vue": "^9.17.0"
```

Comment:
is this a necessary package

Reply:
I had to install it cause the code wasn't running without it.
Might have been something that went wrong just on my local.